src/main/scala/com/typesafe/sbt/packager/debian/NativePackaging.scala

```diff
@@ -43,6 +43,7 @@ trait DebianNativePackaging extends DebianPluginLike {
    * package.
    */
   private[debian] def debianNativeSettings: Seq[Setting[_]] = inConfig(Debian)(Seq(
+    debianNativeBuildOptions += "-Znone", // packages are largely JARs, which are already compressed
```

Comment:
If this is a bit too much refactoring  for you, I can adjust that after merging.